src/partials/messenger.conversation.html

```diff
@@ -22,7 +24,28 @@
                 <span ng-bind-html="ctrl.receiver.members | idsToNames | escapeHtml | emojify"></span>
             </div>
         </div>
+
+        <!-- Menu -->
+        <div class="header-buttons">
```

Comment:
I would prefer this being an unordered list that grows to the left but we can do that when we add more items.